kube-runtime/src/watcher.rs

```diff
@@ -142,8 +142,8 @@ async fn step_trampolined<K: Meta + Clone + DeserializeOwned + Send + 'static>(
                     stream,
                 })
             }
-            Some(Ok(WatchEvent::Bookmark(obj))) => (None, State::Watching {
-                resource_version: obj.resource_ver().unwrap(),
+            Some(Ok(WatchEvent::Bookmark(bm))) => (None, State::Watching {
+                resource_version: bm.version().clone(),
```

Comment:
Nit: Not saving much here over just moving `bm.metadata.resource_version`.

Reply:
it's true, it was just in a few places. happy to remove the impl on Bookmark though